kodilla-hibernate/src/test/java/com/kodilla/hibernate/tasklist/dao/TaskListDaoTestSuite.java

```diff
@@ -0,0 +1,36 @@
+package com.kodilla.hibernate.tasklist.dao;
+
+import com.kodilla.hibernate.tasklist.TaskList;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit4.SpringRunner;
+
+import java.util.List;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest
+public class TaskListDaoTestSuite {
+
+    @Autowired
+    private TaskListDao taskListDao;
+    private static final String NAME = "Submodule 13";
```

Comment:
w testach nie tworz stalych dla danych testowych

Reply:
Ok. Jak zwykle, tak to robili w przykładzie w kursie ;)